org/ystia/experimental/consul/pub/types.yaml

```diff
@@ -23,7 +23,7 @@ metadata:
 
 imports:
   - tosca-normative-types:1.0.0-ALIEN20
-  - org.alien4cloud.consul.pub:2.1.0-SM7
+  - org.alien4cloud.consul.pub:2.1.0-SNAPSHOT
```

Comment:
Should keep the version to alien4cloud 2.1.0-SM7 here, as the change of an alien4cloud version must be done globally on the forge for all components